app/controllers/users/two_factor_authentication_setup_controller.rb

```diff
@@ -43,8 +43,9 @@ def two_factor_options_presenter
     end
 
     def process_valid_form
-      user_session[:selected_mfa_options] = @two_factor_options_form.selection
-      redirect_to confirmation_path(user_session[:selected_mfa_options].first)
+      user_session[:mfa_selections] = @two_factor_options_form.selection
+      user_session[:suggest_second_mfa] = user_session[:mfa_selections].count == 1
```

Comment:
What do you think about not saving a separate key in the session for `:suggest_second_mfa` and just adding a method that checks the count of `user_session[: mfa_selections]`?

```
def suggest_second_mfa?
  user_session[:mfa_selections].count == 1
end